src/lib/isBefore.js

```diff
@@ -1,9 +1,12 @@
 import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
-  const original = toDate(str);
-  return !!(original && comparison && original < comparison);
+export default function isBefore(date, options = {}) {
+  assertString(date);
+
+  const { comparisonDate = Date().toString() } = options;
```

Comment:
Although this is a clean solution, unfortunately it's a breaking change

Reply:
Thanks for the feedback. Could you elaborate, please? :)

The original implementation of this function was different compared to `isAfter`, in that it did not have a legacy mode using the `arguments` object. Is it this you're alluding to above?

Reply:
That is indeed where I'm refering to. `isBefore` should use the `arguments` object in order to be backwards compatible. The goal of my refactor is to support both options and not include breaking changes. If you look at the state of `isAfter` in the master branch you will see that `isAfter` and `isBefore` are basically the same. See the original tests as well

Reply:
Yeap, sorry for that. Already working on it :).